aws/lambda/lambda_function.py

```diff
@@ -29,8 +29,8 @@ def handler(event, context):
     logging.debug(f"Article authors: {article.authors}")
     
     # Send error response if the given URL does not contain a valid article
-    # https://github.com/codelucas/newspaper/blob/master/newspaper/article.py#L322
-    if not article.is_valid_body():
```

Comment:
How about we keep this `is_valid_body()` check and only perform an additional `is_valid_url()` for cases that fail the first check?

Reply:
I made the condition a little more lenient